src/components/Profile/ProfileImage/ProfileImage.tsx

```diff
@@ -0,0 +1,20 @@
+import "./ProfileImage.css";
+import ProfilePicture from "../../../assets/profile.png";
+
+type ImageOptions = {
+  circular?: boolean;
+};
+
+const ProfileImage = (options: ImageOptions = {}) => {
+  const classes = `profile__header__image ${
+    options.circular ? "profile__header__image__circular" : ""

```

Comment:
What do you think about refactoring this using [clsx](https://www.npmjs.com/package/clsx)?

If you prefer you can do this in a separate PR.